setup.cfg

```diff
@@ -21,6 +21,9 @@ ignore =
     E231,
     # not pep8, black adds line break before binary operator
     W503,
+    # Google Python style is not RST until after processed by Napoleon
+    # See https://github.com/peterjc/flake8-rst-docstrings/issues/17
+    RST201,RST203,RST301,
```

Comment:
I thought all these had to be on their own lines since a few versions ago?

Reply:
Seems that this is allowed now:

https://flake8.pycqa.org/en/latest/user/configuration.html#configuration